syntax = "proto3";

package orderexecutor;

service OrderExecutorService {
  rpc ProcessOrder (ProcessOrderRequest) returns (ProcessOrderResponse) {}
}

message ProcessOrderRequest {}

message ProcessOrderResponse {
  bool success = 1;
  string order_json = 2;
  string order_result = 3;
}
